microcosm_elasticsearch/tests/fixtures.py

```diff
@@ -72,3 +78,44 @@ def __init__(self, graph):
 class PlayerStore(Store):
     def __init__(self, graph):
         super(PlayerStore, self).__init__(graph, graph.example_index, Player, graph.example_search_index)
+
+
+@binding("first_attribute_search_index")
+def create_first_attribute_search_index(graph):
+    return PersonSearchIndex(
+        graph=graph,
+        index=graph.first_attribute_index
+    )
+
+@binding("person_overloaded_store")
+class PersonOverloadedStore(Store):
+    def __init__(self, graph):
+        super(PersonOverloadedStore, self).__init__(graph, graph.first_attribute_index, Person, graph.first_attribute_search_index)
```

Comment:
nit: you can also omit the current class name and get the same behaviour in Python3, with just `super().__init__(graph, ...)`

Reply:
Thanks. [PEP-3135](https://www.python.org/dev/peps/pep-3135/)